src/components/LazyComponent.tsx

```diff
@@ -0,0 +1,6 @@
+import { Suspense } from 'react'
+import Loading from 'icons/Loading'
+
+export default function ({ lazyImported }: { lazyImported: JSX.Element }) {
+  return <Suspense fallback={<Loading screenCentre />}>{lazyImported}</Suspense>
```

Comment:
Can you show a few screenshots of how it looks like, please?

Reply:
https://user-images.githubusercontent.com/49658988/169561990-67cb2d7f-0d48-4831-80d9-190f359aaaa1.mp4